package/kernel/linux/modules/ipmi.mk

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (C) 2022 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See https://www.gnu.org/licenses/gpl-2.0.txt for more information
+#
+
+IPMI_MENU:=IPMI system interface support
+
+define KernelPackage/ipmi-handler
+  SUBMENU:=$(IPMI_MENU)
+  TITLE:=IPMI top-level message handler
+  KCONFIG:=CONFIG_IPMI_HANDLER=m \
+           CONFIG_ACPI_IPMI=m \
+           CONFIG_IPMI_PANIC_EVENT=y \
+           CONFIG_IPMI_PANIC_STRING=y \
+           CONFIG_IPMI_DEVICE_INTERFACE=n \
+           CONFIG_IPMI_SI=n \
```

Comment:
Hm, can you run a `make kernel_oldconfig` and see if your target's config-5.10 has new entries with something like

    # CONFIG_IPMI_* not set
    # CONFIG_* not set

and then put all these "disablers" into the target/linux/generic/config-5.10

Reply:
Hmm, did not see any of these changes you mentioned.
Maybe I did something wrong?
Could you explain more how to do?